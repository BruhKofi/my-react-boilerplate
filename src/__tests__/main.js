import React from 'react';
import renderer from 'react-test-renderer';
import Main from '../main';

test('Main', () => {
  const tree = renderer.create(<Main />).toJSON();

  expect(tree).toMatchSnapshot();
});
