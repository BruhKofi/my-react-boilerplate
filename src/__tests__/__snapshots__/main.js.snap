// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main 1`] = `
<div
  className="main"
>
  <div
    className="overlay welcome"
  >
    <h1>
      Welcome.
    </h1>
    <p>
      Press anywhere to increase the counter!
    </p>
  </div>
  <div
    className="clicker"
  >
    <button
      onClick={[Function]}
    >
      Increment
    </button>
    <p
      className="counter"
    >
      0
    </p>
  </div>
  <footer>
    <p>
      Change your 
      <code>
        app.js
      </code>
       file.
    </p>
  </footer>
</div>
`;
